libraries/botbuilder-dialogs-adaptive/src/actions/switchCondition.ts

```diff
@@ -54,6 +54,9 @@ export interface SwitchConditionConfiguration extends DialogConfiguration {
     disabled?: BoolProperty;
 }
 
+/**
```

Comment:
Is this empty comment OK here? Just double checking you didn't miss some content.

Reply:
The class should have a description